ice-phishing-detection-bot/README.md

```diff
@@ -0,0 +1,39 @@
+# Ice Phishing Detection Bot
+
+## Description
+
+This bot detects high approvals for a specific account and if any assets are transfered
+
+## Supported Chains
+
+- Ethereum
```

Comment:
any reason why avalanche is excluded?

Reply:
They don't have an API that would allow us to detect if a contract is verified, will produce too much noise